components/label.json

```diff
@@ -2,35 +2,22 @@
   "label": {
     "comment": "Applied to non editable pieces of text.",
     "primary": {
-      "#normal": {
-        "text": "@theme-text-primary-normal"
-      },
-      "#disabled": {
-        "text": "@theme-text-primary-disabled"
-      }
+      "text": "@theme-text-primary-normal"
```

Comment:
why do you want to take out the #ui-state from these tokens? is it really needed?

Reply:
I want to remove the #ui-state from these tokens because I can't use it on `macOS`. I need the disabled state as an independent state, just like `primary` or `secondary` are defined.

Reply:
Pushed commit
[Reverted label changes](https://github.com/momentum-design/tokens/pull/212/commits/68e0bce65663e5a02fff04da65163dee7b51e904)
Changing this conversation to Resolved as it no longer applies.